Solutions/TenableIO/Data Connectors/TenableAssetExportStatusAndSendChunks/__init__.py

```diff
@@ -19,6 +19,16 @@ def send_chunks_to_queue(exportJobDetails):
     if len(chunks) > 0:
         assets_table = ExportsTableStore(connection_string, assets_table_name)
         for chunk in chunks:
+            chunk_dtls = assets_table.get(exportJobId, str(chunk))
+            if chunk_dtls:
+                current_chunk_status = chunk_dtls['jobStatus']
+                if (
+                    current_chunk_status == TenableStatus.sent_to_queue.value or
+                    current_chunk_status == TenableStatus.finished.value
+                ):
+                    logging.warn(f'Avoiding asset chunk duplicate processing -- {exportJobId} {chunk}. Current status: {current_chunk_status}')

```

Comment:
Line changes look good. Just few small suggestions-
- Would you please use logging.warning() instead of logging.warn() ? Because  logging.warn() is deprecated. 
- And please format these lines like below if it does not create any problem.

```
            if chunk_dtls:
                current_chunk_status = chunk_dtls['jobStatus']
                if (
                        current_chunk_status == TenableStatus.sent_to_queue.value or
                        current_chunk_status == TenableStatus.finished.value
                ):
                    logging.warn(
                        f'Avoiding asset chunk duplicate processing -- {exportJobId} {chunk}. '
                        f'Current status: {current_chunk_status}')
                    continue
```
Apply same things [Solutions/TenableIO/Data Connectors/TenableVulnExportStatusAndSendChunks/__init__.py](https://github.com/tenable/Azure-Sentinel/pull/4/files#diff-f0efab32e9a09e638f103f839d3552932c5360528e1b654882c2c662433cea07) as well.

Reply:
@MitaliUpadhyay All review comments addressed.